### PR TITLE
[FIXED] After search results of user the category of search should still remain 'Users' and not projects

### LINKED ISSUE #1667: After search results of user the category of search should still remain 'Users' and not projects
**Describe the bug**
when a user is searched it shows result of  all registered users but category of search automatically changes to projects  

**To Reproduce**
1. Go to 'www.circuitverse.org'
2. Click on 'search tab and search for user'
3. after search of user the  category shows projects as category and not users

**Expected behavior**
  If a user is searched, after search results category of search should remain users only and not projects
 and if projects are searched then category of search should still remain projects

**Screenshots**

as shown in screenshot we have results of user search  but category of search gets changed to projects automatically 

Screenshot from 2020-09-19 17-25-28

**Are you working on this issue? (Yes/No)**
No

### PR DESCRIPTION
**Pull request by DarthCucumber**
Fixes #1667 

#### Describe the changes you have made in this PR -
Added a script which retains the search filter applied to the search from **_select tag_**.

### Screenshots of the changes (If any) -
before:
if  **Users** filter is selected from **options** (_select tag_) and a name is searched, the **option** (_select tag_)  in the results page reverts to default value rather than staying **Users**.
bug

after:
after fixing it looks like so:
fix

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -31,7 +30,6 @@
             <% end %>
           </div>
           <div class="col-6 footer-links">
-            <h6>Info</h6>
```
Comment (tachyons) [inline app/views/layouts/_footer.html.erb:34]: Unrelated

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -90,4 +90,20 @@
   if (window.location.href.includes('search')) {
       searchBar.style.display = "block";
   }
+
+    /* for retaining select tag search query after a search*/
+  if(window.location.pathname==="/search"){
```
Comment (tachyons) [inline app/views/layouts/_header.html.erb:95]: This seems more like a workaround than a fix, ideally properly configured form should be handling this scenario itself

Comment (DarthCucumber) [reply app/views/layouts/_header.html.erb:95]: Okay, I'll make the changes accordingly and make another pull request. Thanks for the feedback.

File: app/views/layouts/_footer.html.erb (hunk 1 above)
Comment (DarthCucumber) [reply app/views/layouts/_footer.html.erb:34]: sorry about this. 
This is the previous commit. somehow it got attached. I'll fix it.

Comment (DarthCucumber) [reply app/views/layouts/_footer.html.erb:34]: @tachyons I made changes to the form and this time it's not a workaround. When I'm making a pull request the above commits I made previously are also adding up to the current one. Can you please help me.
